tests/Operation/WatchFunctionalTest.php

```diff
@@ -320,6 +379,54 @@ public function testMaxAwaitTimeMS()
         $this->assertTrue($changeStream->valid());
     }
 
+    public function testResumeAfterKillThenNoOperations()
+    {
+        $operation = new Watch($this->manager, $this->getDatabaseName(), $this->getCollectionName(), [], ['maxAwaitTimeMS' => 100]);
+        $changeStream = $operation->execute($this->getPrimaryServer());
+
+        $this->insertDocument(['_id' => 1, 'x' => 'foo']);
```

Comment:
I think we can simply omit this line. Asserting that `rewind()` doesn't throw an exception is enough to test that it can resume after not finding a cursor.

Reply:
This comment was resolved. No longer applicable to whatever line this points to.